### ISSUE TITLE
Missing column in earnings table

### ISSUE THREAD

**Issue by philnorc** [Greenstand/treetracker-earnings-api#66]
According to testCases.md, 
https://github.com/Greenstand/treetracker-earnings-api/blob/main/docs/test/testCases.md,
the earnings table should have a column 
"Sub organization name"
Neither the test panel nor the dev panel have that column.
https://test-admin.treetracker.org/earnings
https://dev-admin.treetracker.org/earnings

**Comment by dadiorchen** [comment]
The same as #67

**Comment by philnorc** [comment]
No. 66 & 67 are not the same.
#67 concerns the column "Count of Captures" in the payments table.
#66 concerns the column "Sub organization" in the earnings table.

**Comment by philnorc** [comment]
The missing column for "Organization" is particularly a problem because we provide a filter option for Organization. It is hard to use a filter for values you cannot see. 

And where do the values in the Organization menu come from? They do not include the only organizations I see in the data: Child One, Child Two, Greenstand, and null.

**Comment by dadiorchen** [comment]
Yes, I agree, I added this feature to milestone 2, because adding this feature cause change on both back end and front end, it's risky to add it at this testing round stage.

The org menu is from the infrastructure of the admin panel, there are some problems with the authentication workflow, so you find it's incorrect, we will figure it out in the future, and also it's not the problem of earning tool (it's about stakeholder system)
